GVFS/GVFS.Platform.POSIX/POSIXDiskLayoutUpgradeData.cs

```diff
@@ -1,27 +1,27 @@
-﻿using GVFS.Common;
-using GVFS.DiskLayoutUpgrades;
-
-namespace GVFS.Platform.Mac
-{
-    public class MacDiskLayoutUpgradeData : IDiskLayoutUpgradeData
-    {
-        public DiskLayoutUpgrade[] Upgrades
-        {
-            get
-            {
-                return new DiskLayoutUpgrade[0];
-            }
-        }
-
-        public DiskLayoutVersion Version => new DiskLayoutVersion(
-                    currentMajorVersion: 18,
-                    currentMinorVersion: 0,
-                    minimumSupportedMajorVersion: 18);
-
-        public bool TryParseLegacyDiskLayoutVersion(string dotGVFSPath, out int majorVersion)
-        {
-            majorVersion = 0;
-            return false;
-        }
-    }
-}
+using GVFS.Common;
+using GVFS.DiskLayoutUpgrades;
+
+namespace GVFS.Platform.POSIX
+{
+    public class POSIXDiskLayoutUpgradeData : IDiskLayoutUpgradeData
```

Comment:
I think the `IDiskLayoutUpgradeData` implementations will need to stay platform specific.

Especially as Linux and Mac are being built up I expect that we'll need to update the disk layout on one platform without impacting the other (e.g. we just had to bump Mac from 17 to 18 while leaving Windows at 17).

If there ends up being common `DiskLayoutUpgrade` steps those can be shared, but I think that the major/minor disk layout versions will need to be kept separate.

Reply:
Reverted in e67c4efb.